pucauto.py

```diff
@@ -264,66 +276,71 @@ def build_trades_dict(soup):
         if trades.get(member_id):
             # Seen this member before in another row so just add another card
             trades[member_id]["cards"].append(card)
+            trades[member_id]["value"] += card_value
```

Comment:
For performance, sum the value as we go along instead of iterating through the structure later.